### PR TITLE
Adding pinot configs to trino-server-dev

### PR DESCRIPTION
**Pull request by xiangfu0**
Adding pinot config into trino test server

### CODE REVIEW

File: testing/trino-server-dev/etc/catalog/pinot.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#
```
Comment (ebyhr) [inline testing/trino-server-dev/etc/catalog/pinot.properties:1]: What's the motivation of this change? There's no product test environment for Pinot at this time.

Comment (xiangfu0) [reply testing/trino-server-dev/etc/catalog/pinot.properties:1]: I'm using this to test the connectivity with a local Pinot deployment. What do you mean by a test environment?

Comment (xiangfu0) [reply testing/trino-server-dev/etc/catalog/pinot.properties:1]: If you can point me to an example, I can add a pinot test env as well

Comment (ebyhr) [reply testing/trino-server-dev/etc/catalog/pinot.properties:1]: "product test" means https://github.com/trinodb/trino/tree/master/testing/trino-product-tests https://github.com/trinodb/trino/commit/10909a411698f611ebf957c5400963c95c8e64cd is the example when we added a ClickHouse product test.
